### ISSUE TITLE
My message to @cuekYT

### ISSUE THREAD

**Issue by ymeedo** [playingoDEERUX/growbrewproxy#34]
I have a few questions for you.

- Why does GitHub say you joined 12 hours ago?
- Why was this repository just made and has very few commits, yet it's at version 2?
- Why do you call people "mentally disabled slaves of a disabled retard"? You should be more considerate to people with mental disabilities.
- @playingoDEERUX's repository actually has way more stars and a deeper history of commits, yet, your source code and his looks exactly the same, which I consider proof enough that this indeed @playingoDEERUX. Why are you denying this?
- I've seen a few issues from iProgramMC commenting here, and they have all been deleted. I've also seen in @playingoDEERUX's issues, that his issue was edited to "im slave" "i suck playingos dick" which is funny to me, because you seem to be very immature. Why do you behave like this?
- Why do you keep deleting messages? Are you YandereDev 2.0?

TL;DR: Your allegations that @playingoDEERUX stole your code are proven false, thus, you can expect a DMCA takedown notice in the next 168 hours.

**Comment by ymeedo** [comment]
@cuekYT

**Comment by playingoDEERUX** [comment]
100% true message, validated and confirmed.

**Comment by iProgramMC** [comment]
💯💯VALIDATED💯💯✅✅❎☑️✔️🏁

**Comment by playingoDEERUX** [comment]
💯💯VALIDATED💯💯✅APPROVED✅❎☑️✔️🏁CONFIMED🤪😂😯

**Comment by CuekGaming** [comment]
Remember, im not CuekYT

**Comment by playingoDEERUX** [comment]
Closing, hope trolls like these (cuekYT) don't appear again.
